### DISCUSSION TITLE
How to disable --parallel job specification

### DISCUSSION THREAD

**Discussion by timharry** [microsoft/vscode-cmake-tools#3668] [Q&A]
It documentation says to set the parallel jobs value to 1 for this to not be passed into the arguments of the generator... But when I set it to 1, my build command looks like:

`cmake --build buildDir --parallel 1 --target all`

This obviously sets the job number to 1, instead of not specifying to at all. 

How can I get this to not specify the job number? I'm using ninja as my generator, and my cmake tools version 1.17.17.

**Comment by gcampbell-msft** [comment]
@timharry Taking a look at the code, the suggestion you mention only applies to msbuild generators. Otherwise, we either use what you set in that setting, or we have a default that we grab based on the number of CPUs. https://github.com/microsoft/vscode-cmake-tools/blob/ec03fc1efbc5e62fb3666b4fc1dde2c1024c4e4c/src/config.ts#L18

**Comment by gcampbell-msft** [comment]
To help us understand whether you'd like this as a feature request, what is your scenario such that you don't want to specify this in the command-line? 

If you'd like this as a feature, please create an issue with the details so that we can track it better. Thanks!
